### PR TITLE
Create signalfxexporter README.md

### PR DESCRIPTION
**Pull request by flands**
**Documentation:** Add signalfx exporter documentation

### CODE REVIEW

File: exporter/signalfxexporter/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# SignalFx Metrics Exporter
+
+How to send metrics to SignalFx.
+
+The following configuration options are required:
+
+- `access_token` (no default): AccessToken is the authentication token provided
+by SignalFx or another backend that supports the SAPM proto.
```
Comment (pjanotti) [inline exporter/signalfxexporter/README.md]:
```suggestion
by SignalFx or another backend that can receive SignalFx metrics.
```

Comment (pjanotti) [reply exporter/signalfxexporter/README.md]: alternatively, just remove the part starting after "by SignalFx"

File: exporter/signalfxexporter/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- `realm` (default = us0): SignalFx realm where the data will be received.
+
+The following configuration options can also be configured:
+
+- `headers` (no default): Headers to pass in payload
```
Comment (pjanotti) [inline exporter/signalfxexporter/README.md]:
```suggestion
- `headers` (no default): Headers to pass in the payload.
```

File: exporter/signalfxexporter/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- `timeout` (default = 5s): Amount of time to wait for send operation to complete.
```
Comment (pjanotti) [inline exporter/signalfxexporter/README.md]:
```suggestion
- `timeout` (default = 5s): Amount of time to wait for a send operation to complete.
```

**Review by pjanotti — COMMENTED** [review]
just a few suggestions, otherwise LGTM

File: exporter/signalfxexporter/README.md (hunk 1 above)
Comment (flands) [reply exporter/signalfxexporter/README.md]: Removed rest

**Review by pjanotti — APPROVED** [review]
LGTM

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**